formatters/swiftcov/parse.go

```diff
@@ -0,0 +1,128 @@
+package swiftcov
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codeclimate/test-reporter/env"
+	"github.com/codeclimate/test-reporter/formatters"
+	"github.com/markbates/pop/nulls"
+	"github.com/pkg/errors"
+)
+
+// Formatter collects SwiftCov files, parses them, then formats them into a single report.
+type Formatter struct {
+	FileNames   []string
+	SourceFiles []formatters.SourceFile
+}
+
+var searchPaths = []string{"./"}
+var search = ".swift.gcov" // look for these file extensions
```

Comment:
Do you know if there is anything different about SwiftCov than other tools that use gcov? If we structured this as a "gcov formatter" would it work for Swift as well as other tools that use gcov (C/C++ come to mind right away)